addons/sourcemod/scripting/surftimer/globals.sp

```diff
@@ -1152,6 +1152,12 @@ char g_szCountry[MAXPLAYERS + 1][100];
 char g_szCountryCode[MAXPLAYERS + 1][3];
 char g_szContinentCode[MAXPLAYERS + 1][3];
 
+// mappers name
+char g_szMapperName[32];
+
+// mapper name null?
+bool g_bMapperNameFound;
```

Comment:
You could check the string length instead of this bool

Reply:
maybe, but i don't see why we can't just use the bool